test/pr-review.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash -xe
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o xtrace
+
+usage() {
+  echo "Usage: $0 PR_NUMBER" 1>&2;
+  exit 1;
+}
+
+if [ -z "${PULL_NUMBER}" ]; then
+  if [ -z "$1" ];then
+    usage
+  fi
+  PR_NUMBER=$1
+fi
+
+if [ -z "${VERIFICATION_PAUSE}" ]; then
```

Comment:
This line gives me the error: `./test/pr-review.sh: line 20: VERIFICATION_PAUSE: unbound variable` so the default doesn't get set.